### PR TITLE
[UN-714] Fix position of 'new' label for secondary card links and adjust padding

### PR DESCRIPTION
**Pull request by janehughes**
Ticket URL: https://national-archives.atlassian.net/browse/UN-714

## About these changes

Adjusts the green 'new' label assigned to new stories/records in the secondary cards - makes sure the label doesn't drop below the title when the title is longer than 1 line.

## How to check these changes

Review /explore-the-collection/ locally, check a secondary card with a long title and see that the 'new' label stays on the same line as the title and doesn't drop to a new line. 

Before: 
image

After:
image

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [ ] Waited for all CI jobs to pass before requesting a review.
- [ ] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: sass/includes/_card-groups.scss [hunk 1]
```diff
@@ -71,14 +69,18 @@
...
 
     &__heading {
-      display: inline;
+      display: block;
+      margin-top: 0.3rem;
+      margin-bottom: 0.5rem;
 
       span {
         background: green;
+        display: inline;
```
Comment (ahosgood) [inline sass/includes/_card-groups.scss:78]: `display: inline;` won't honour padding - you'll have to use `display: inline-block;`

**Review by jamesbiggs — APPROVED** [review]
Hi Jane, nice work - this works for me but I see AJ has left a comment, so happy to approve but also take a look at AJ's comment :) I've tested with just inline and also inline-block and both work in my eyes.

File: sass/includes/_card-groups.scss (hunk 1 above)
Comment (janehughes) [reply sass/includes/_card-groups.scss]: 💯  good spot thanks - I've updated to inline-block and also adjusted the padding ever so slightly to accommodate the line height of the text better. Such a small change it's hardly worth it :)

**Comment by janehughes** [comment]
> Hi Jane, nice work - this works for me but I see AJ has left a comment, so happy to approve but also take a look at AJ's comment :) I've tested with just inline and also inline-block and both work in my eyes.

Thanks James I've updated it now to inline-block to support the padding better and avoid any possible cross-browser weirdness - if you're happy to merge please go for it :)

**Pull request merged**